sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -89,25 +89,14 @@ there are non-existent document IDs also passed through the API - See [PR 35513]
   JVM configuration - `COSMOS.MIN_CONNECTION_POOL_SIZE_PER_ENDPOINT` - See [PR 34859](https://github.com/Azure/azure-sdk-for-java/pull/34859)
 * Extending maximum retry delay in `SessionTokenMismatchRetryPolicy`. - See [PR 35360](https://github.com/Azure/azure-sdk-for-java/pull/35360)
 
-### 4.45.2-hotfix (2023-09-18)
```

Comment:
same here

Reply:
Faulty merge conflict resolution. Fixing this in next iteration.